irc/server.go

```diff
@@ -204,6 +209,37 @@ func (s *Server) listen(addr string) {
 }
 
 //
+// websocket listen goroutine
+//
+
+func (s *Server) wslisten(addr string) {
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
not super easy to listen with multiple http handler on different port, I need to check if I can do that without too many code.
We can add that for in a new PR